generator/java/build.gradle

```diff
@@ -33,13 +36,21 @@ task generateClient(type: GradleBuild) {
     def taskList = ['cleanPreviousJavaBuild'];
 
 	rootProject.swaggerSourceList.each {
-        def apiName = it.getName().replace(".json", "").toLowerCase();
         def jsonFile = file(it.path);
         def parsedJson = new groovy.json.JsonSlurper().parseText(jsonFile.text);
-        def clientOutputDirPerVersion = clientOutputDir + "/" + it.getName().replace(".json", "");
+
+        def parsedApiName = it.getName().replace(".json", "").toLowerCase();
+        def (criteoService, apiVersionFromName) = parsedApiName.tokenize('_');
+        def criteoApiVersion = apiVersionFromName.replace('-', '_');
+        
+        def clientOutputDirPerVersion = clientOutputDir + "/" + parsedApiName;
         def specPath = it.path;
+        def packageBase =  criteoApiVersion; // com.criteo.api.marketingsolutions.v2021_10
```

Comment:
I am not sure the comment is really accurate

Reply:
The comment should be  on line 51

Reply:
Is the comment showing an example of the actual state of a variable or its final value?